### DISCUSSION TITLE
clip-start-time, youtube, bubble.io

### DISCUSSION THREAD

**Discussion by abvas** [vidstack/player#1167] [Q&A]
This may not be directly related to the plugin, but I need advice on using the "**clip-start-time**" parameter for **youtube** videos.
I'm trying to use **youtube** videos and "**clip-start-time**" on the "**bubble.io**" platform and the video always starts automatically even if the "**autoplay**" option is not set. This only happens with videos on **YouTube**, but for other providers everything works fine and the video does not start automatically. Also on other platforms, including locally and on https://codepen.io/ everything is fine and the video also does not start automatically.

I use the simplest connection diagram:
```
<head>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/vidstack@^1.0.0/player/styles/default/theme.min.css"/>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/vidstack@^1.0.0/player/styles/default/layouts/video.min.css"/>
<script src="https://cdn.jsdelivr.net/npm/vidstack@^1.0.0/cdn/with-layouts/vidstack.js" type="module"></script> 
</head>

<media-player src='https://www.youtube.com/watch?v=ERjt6t5Rus0' clip-start-time='10'>
  <media-provider>
    <media-poster class='vds-poster'></media-poster>
  </media-provider>	
  <media-video-layout></media-video-layout>
```

Do you have any ideas why this might be happening and what can be done about it.

Thank you.

**Comment by mihar-22** [comment]
The YouTube embed unfortunately autoplays on the initial seek (internally). We have some logic to prevent it but it's not perfect and hard to catch.
